src/promptflow/promptflow/contracts/trace.py

```diff
@@ -8,13 +8,42 @@
 
 
 class TraceType(str, Enum):
+    """An enumeration of possible trace types.
+
+    :cvar LLM: The LLM trace type.
+    :cvar TOOL: The tool trace type.
```

Comment:
Pleause use `:param` and `:type` instead `:cvar` to keep consistent with other docs

Reply:
Could you help me understand `rendered effect`? I am little bit confused on that. :(

Reply:
I meant the effect on the built doc site:
https://microsoft.github.io/promptflow/reference/python-library-reference/promptflow.html

Noticed you have deleted the cvar parameters, may ignore this comment.